curvefs/src/mds/metric/metric.h

```diff
@@ -59,15 +59,26 @@ class FsMountMetric {
     // current number of fs mountpoints
     bvar::Adder<int64_t> count_;
 
-    using MountPointMetric =
-        std::unordered_map<std::string,
-                           std::unique_ptr<bvar::Status<std::string>>>;
+    using MountPointMetric = std::unordered_map<std::string,
+        std::unique_ptr<bvar::Status<std::string>>>;
     // protect mps_
     Mutex mtx_;
 
     MountPointMetric mps_;
 };
 
+struct FsUsageMetric {
```

Comment:
maybe fs capacity is also need like fs usage

Reply:
> maybe fs capacity is also need like fs usage

fix